src/assets/src/components/home.tsx

```diff
@@ -47,7 +47,7 @@ export function HomePage(props: PageProps) {
             <QueueLookup/>
             <hr/>
             <Link to="/manage">
-                Manage Your Own Queues
+                Manage Queues
             </Link>
```

Comment:
While it's not always possible, keeping a PR focused on one issue is preferred. This change for #219 involves different files/functionality, so it's a good candidate to be separated out. This would help us keep track of and QA changes as we prepare for a release. If you'd like to open a separate PR with this change, I will review it. If you aren't able to, I can also cherry-pick your commit here and submit it separately.